libraries/tools/kotlin-gradle-plugin/src/common/kotlin/org/jetbrains/kotlin/gradle/testing/internal/KotlinTestsRegistry.kt

```diff
@@ -72,7 +72,7 @@ class KotlinTestsRegistry(val project: Project, val allTestsTaskName: String = "
             aggregate.description = description
             aggregate.group = JavaBasePlugin.VERIFICATION_GROUP
 
-            aggregate.destinationDir = project.testReportsDir.resolve(reportName)
+            aggregate.destinationDirectory.set(project.testReportsDir.resolve(reportName))
```

Comment:
`TestReport.destinationDirectory` input was added only in Gradle 7.4. Since KGP supports all Gradle versions from 6.8.3 I could not accept this change.

I need to think how to fix it better.